startup_assistance/indico_startup_assistance/migrations/20190220_1138_b160812c186b_add_startup_assistance_requests_table.py

```diff
@@ -0,0 +1,68 @@
+"""Add startup assistance requests table
+
+Revision ID: b160812c186b
+Revises:
+Create Date: 2019-02-20 11:38:08.691528
+"""
+
+from __future__ import unicode_literals
+
+import sqlalchemy as sa
+from alembic import context, op
+from sqlalchemy.sql.ddl import CreateSchema, DropSchema
+
+
+# revision identifiers, used by Alembic.
+revision = 'b160812c186b'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if context.is_offline_mode():
+        raise Exception('This upgrade is only possible in online mode')
+
+    op.execute(CreateSchema('plugin_startup_assistance'))
+    op.create_table(
+        'startup_assistance_requests',
+        sa.Column('reservation_id', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(['reservation_id'], ['roombooking.reservations.id']),
+        sa.PrimaryKeyConstraint('reservation_id'),
+        schema='plugin_startup_assistance'
+    )
+
+    conn = op.get_bind()
+    res = conn.execute('''
```

Comment:
i think you can turn this into one query that does the insert for each matching row. that way you can even use `op.execute()` and don't need to force online mode